_includes/tutorials/joining-stream-table/ksql/markup/dev/create-inputs.adoc

```diff
@@ -2,4 +2,6 @@ First, you'll need to create a Kafka topic and table to represent the movie refe
 
 +++++
 <pre class="snippet"><code class="sql">{% include_raw tutorials/joining-stream-table/ksql/code/tutorial-steps/dev/create-movies-table.sql %}</code></pre>
-+++++
\ No newline at end of file
++++++
+
+In this case the lookup data originates from a Kafka topic but this doesn't always have to be the case. You can use Kafka Connect to stream data from a source system (such as a database) into a Kafka topic, which could then be the foundation for your ksqlDB lookup table. For further reading checkout this tutorial on link:https://kafka-tutorials.confluent.io/connect-add-key-to-source/ksql.html[creating a ksqlDB table from PostgresSQL data using Kafka Connect].
```

Comment:
@awalther28 awesome text.  Given that there's also a `kstreams` KT (for both this KT and the JDBC KT), maybe this text should also be added to `_includes/tutorials/joining-stream-table/kstreams`?  (with slight modification `:%s/ksqlDB/Kafka Streams/`)

Reply:
@ybyzek I added a similar blurb in the kstreams example in commit [84cad85](https://github.com/confluentinc/kafka-tutorials/pull/649/commits/84cad85fe3c8934fbdf48eb042f1dc161e6e24f7).